app/Http/Resources/ServiceResource.php

```diff
@@ -24,14 +25,13 @@ public function toArray($request)
         return [
             'id' => $this->id,
             'user_id' => $this->user_id,
+            'user' => User::find($this->user_id),
```

Comment:
Me ocorreu que na PR #105 o usuário criador dos itens seria sempre o usuário autenticado, pois estava sendo utilizado o `Auth::user()`. Eu corrigi isto inserindo o usuário diretamente no objeto retornado pelos _endpoints_ `/api/feedbacks` e `/api/services`, mas essa solução parece meio gambiarra, caso haja sugestões para soluções melhores será ótimo.

Reply:
Estou um pouco confuso, por exemplo, caso um novo item for criado o usuário associado à esse item não será sempre o que está autenticado?

Reply:
Isso, mas para mostrar os serviços, caso o usuário for um admin ele poderá ver todos os serviços e pode ser que alguns deles tenham sido criados por outros usuários.